fdb-extensions/src/test/java/com/apple/foundationdb/async/rtree/RTreeModificationTest.java

```diff
@@ -57,6 +57,7 @@
  * Tests testing insert/update/deletes of data into/in/from {@link RTree}s.
  */
 @Tag(Tags.RequiresFDB)
+@Tag(Tags.Slow)
```

Comment:
While running tests locally, I found it expedient to run tests with `SKIP_SLOW_TESTS=true ./gradlew test`. I've marked some tests as `Slow` if they exceeded the 2 second limit that we now apply